src/devices/src/virtio/net/mod.rs

```diff
@@ -14,6 +14,7 @@ pub const TX_INDEX: usize = 1;
 
 pub mod device;
 pub mod event_handler;
+mod iovec;
```

Comment:
nit: Is this used transitively? Not better to be imported in files that specifically need it?

Reply:
These declarations in `mod.rs` are not the equivalent of `include`/`import`. They just declare which files contain code which is part of the module.